practice-one/src/components/introduction.html

```diff
@@ -0,0 +1,24 @@
+<section class="introduction-container">
+  <div class="introduction-content-container">
+     <div class="people-trust-icon">
+        <img src="./assets/images/trust.svg" alt="Heart icon" />
+        People Trust us
+     </div>
+     <h1 class="header-introduction">
+        <b>  
+           We're 
+          <span class="primary-color"> Serious </span>
+           For 
+          <br/>
+          <span class="primary-color">Food</span> 
+           & 
+          <span class="accent-color">Delivery</span>.
+        </b>
+     </h1>
```

Comment:
This is wrong, you shouldn't wrap everything inside b tag like this

Reply:
I understood , thanks for your comment